app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -42,11 +45,15 @@ private fun Button(
     backgroundColor: Color,
     icon: Painter? = null,
     tint: Color,
+    minHeight: Dp? = null,
     onClick: () -> Unit,
 ) {
+    val modifier = Modifier.fillMaxWidth()
```

Comment:
@MozillaNoah I might be imagining things, but I feel like there was a conversation about the heights of these buttons with UX, but maybe that was just for the content padding. 

My general feeling here is that we should add the min height directly into the generic `Button` since this is probably a common accessibility issue for all the buttons. However, I also want to make sure this also aligns with our design components in the Figma https://www.figma.com/file/ujdiReypSypu2GhRgzArea/Android-Components?node-id=1202%3A53176&t=IUCFdcNry2HHaHrd-0. The Figma for the Button component should be updated ideally if the existing design is inaccessible. I would like us to avoid setting the min height for individual buttons if we do know this is a common accessibility issue for all the buttons. Aside I am wondering if the actual height of the button needs to grow or if we could just increase the tap area since we have done this for other XML buttons while maintaining the desired button look from UX.

So, my ask here is to figure this out with UX:
(1) See if we should be increasing the size for all the buttons to be min height of 48dp
(2) Alternatively, increasing the tap area while preserving the existing height
(3) Fix the accessibility issue for all the buttons if this is a common problem among all of them.

Reply:
[Here's the original convo](https://mozilla.slack.com/archives/C03KPTP37BN/p1673481343508669)

I concur it'll likely be one of the three options you've presented. Generally, we want the usages of our components to all be aligned before adding parameters to the API contract which can customize the look/feel but are also optional.

Reply:
@gabrielluong @MozillaNoah thank you both for your input! Closing this issue following discussion with @gabrielluong and request for further input from UX.